jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -10,13 +10,81 @@
 from jdaviz.core.registries import tray_registry
 from jdaviz.core.template_mixin import (PluginTemplateMixin, ViewerSelect, LayerSelect,
                                         PlotOptionsSyncState)
+from jdaviz.core.user_api import PluginUserApi
 from jdaviz.core.tools import ICON_DIR
 
 __all__ = ['PlotOptions']
 
 
 @tray_registry('g-plot-options', label="Plot Options")
 class PlotOptions(PluginTemplateMixin):
+    """
+    The Plot Options Plugin gives access to per-viewer and per-layer options and enables
+    setting across multiple viewers/layers simultaneously.
+
+    Only the following attributes and methods are available through the
+    :ref:`public plugin API <plugin-apis>`:
+
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.show`
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.open_in_tray`
+    * ``multiselect``:
+      whether ``viewer`` and ``layer`` should both be in multiselect mode.
+    * ``viewer`` (:class:`~jdaviz.core.template_mixin.ViewerSelect`):
+    * ``layer`` (:class:`~jdaviz.core.template_mixin.LayerSelect`):
+    * :meth:`select_all`
+    * ``layer_visible`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`)
+    * ``subset_visible`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      whether a subset should be visible.
+    * ``subset_color`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not exposed for Specviz
+    * ``axes_visible`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`)
+    * ``collapse_function`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      only exposed for Cubeviz
+    * ``line_color`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not exposed for Imviz
+    * ``line_width`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not exposed for Imviz
+    * ``line_opacity`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not exposed for Imviz
+    * ``as_steps`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not exposed for Imviz
+    * ``uncertainty`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not exposed for Imviz
+    * ``stretch_func`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not exposed for Specviz
+    * ``stretch_preset`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):

```

Comment:
right, but the advice I got was to try to be consistent with the UI, even when we can't be consistent with glue and/or astrowidgets.  We're restricted in some cases by how glue is designed, but I think we should try to just make this as intuitive as possible and not worry too much about direct parallels.  If a user wants to use astrowidgets API instead, they can of course do that.  I'm trying to keep the prefixes to be fixed with the sections in the plugin and so all "stretch" options all share the "stretch" prefix.

Reply:
Strictly speaking, this is not really a "stretch" property in the first place? It defines the display cut off values, hence "cuts". Stretch, I think, is the function used to "stretch out" the values within cuts?

Reply:
I think that's just semantics.  "Stretching the input values between XX and YY with a linear/logarithmic/whatever function".  But if cuts is preferred in the UI as well, then I'm not opposed to changing it here (although I still think a common prefix is a nice way to tie these together in the API since they'll then be sorted next to each other in `dir` calls, tab completion, etc).

Reply:
It would deviate from astrowidgets but I am open to opinions from other devs as well.

Reply:
We have to either deviate from glue, astrowidgets, or both (since they already aren't consistent with each other).  The primary objective here though is to provide something that is intuitive and convenient for a new user who may not know either of those two systems and have a 1:1 mapping with the plugin UI, at the cost of slight confusion for advanced users (and devs) who may already know one or both.

Reply:
I thought you said Jenn said to match astrowidgets if possible? Did I misunderstand?

Reply:
She said the priority should be in matching the UI as closely as possible (we can change some labeling in the UI though) so that users can transition between the two as easily as possible.